Assets/Scripts/AIGhost.cs

```diff
@@ -7,17 +7,26 @@ public class AIGhost : MonoBehaviour
     public Transform[] target;
     public float speed;
     private int currTarget;
+    private GameObject playerObject;
+    SanitySystem sanitySystem;
     // Use this for initialization
     void Start()
     {
         currTarget = 1;
+        playerObject = GameObject.Find("Player");
+        if(playerObject != null)
+            sanitySystem = playerObject.GetComponent <SanitySystem>();
     }
 
     // Update is called once per frame
     void Update()
     {
         float step = speed * Time.deltaTime;
         transform.position = Vector3.MoveTowards(transform.position, target[currTarget].position, step);
+        if(currTarget == 0 && playerObject != null)
```

Comment:
should there be also be an sanitySystem.sanity > 0.0f check like on line 18 in Assets/Scripts/SanitySystem.cs